src/main/kotlin/no/nav/k9/los/nyoppgavestyring/ko/OppgaveKoTjeneste.kt

```diff
@@ -120,6 +123,26 @@ class OppgaveKoTjeneste(
         }
     }
 
+    @WithSpan
+    suspend fun hentAntallMedOgUtenReserverteForKø(
+        oppgaveKoId: Long,
+        skjermet: Boolean,
+    ): AntallOppgaverOgReserverte {
+        return coroutineScope {
+            val antallUtenReserverte = async(Dispatchers.IO) {
```

Comment:
her mister opentelemetry kontekst, tror følgende blir riktig syntaks for å bevare denne

async(Dispatchers.IO + Span.current().asContextElement())